giddy/markov.py

```diff
@@ -197,12 +197,14 @@ class Spatial_Markov(object):
                       If true, quantiles are taken over the entire n*t
                       pooled series. If false, quantiles are taken each
                       time period over n.
-    discrete        : bool
-                      If true, categorical spatial lags which are most common
-                      categories of neighboring observations serve as the
-                      conditioning and fixed is ignored; if false, weighted
-                      averages of neighboring observations are used. Default is
-                      false.
+    lag             : str or obj
```

Comment:
Maybe we can keep the parameter `discrete` and add another parameter for user-supplied spatial lags? To allow for a smoother transition from old APIs.

Reply:
I could still make these changes if you're interested in adding this functionality

Reply:
Sure. That would be nice. I am reopening the PR.

Reply:
sweet. battery permitting, I'll see if i can get this done on the next flight